graviti/schema/field.py

```diff
@@ -16,8 +19,43 @@ class Field(NamedTuple):
     name: str
     type: PortexType
 
+    @property
+    def imports(self) -> Imports:
+        """Property imports to indicate the imports of this field.
+
+        Returns:
+            The imports of this field.
+
+        """
+        imports = Imports()
+        imports.update_from_type(self.type)
+        return imports
```

Comment:
Why create a new `Imports` instance each time to get the `imports` property?

Reply:
Because Field is a `NamedTuple` we need to keep it the same behavior with a normal tuple (name, type)